### ISSUE TITLE
Add super-convergence learning rate schedules

### ISSUE THREAD

**Issue by dthiagarajan** [dthiagarajan/ptb-xl-research#5]
- Switch the optimizer being used to AdamW.
- Set up OneCycleLR training
- Set up discriminative learning rates
- Set up gradual unfreezing (benchmark this first on the simple 1D CNN and ResNet-18 to see if it's worthwhile to pursue this)
